dawn/webpack.config.babel.js

```diff
@@ -24,6 +24,7 @@ const plugins = [
   new webpack.DefinePlugin({
     VERSION: JSON.stringify(version),
   }),
+  new webpack.IgnorePlugin(/vertx/),
```

Comment:
Sometimes there's a weird bug when `make watch` stops due to finding a missing item. This was an attempt to prevent that, but it seems there are other bugs.